packages/web-app-external-sites/README.md

```diff
@@ -0,0 +1,32 @@
+# web-app-external-sites
+
+This application can be used for adding external sites to the application menu of ownCloud Web. External sites can either be opened in a new tab/window or embedded within ownCloud Web via an iFrame.
+
+## Configuration
+
+The configuration for the external-sites app may look like so:
+
+```
+"config": {
+  "sites": [
+    {
+      "name": "ownCloud",
+      "url": "https://www.owncloud.com",
+      "target": "external",
+      color: "#0D856F",
+      "icon": "cloud",
+      "priority": 50
+    }
+  ]
+}
+```
+
+The `name` will appear in the app menu, while `url` specifies the url which will be opened or embedded. `target` can either be `external` or `embedded`. Choose `external` if the site should be opened in a new tab or window. Choose `embedded` if the external content should be embedded within ownCloud Web. All of these 3 config options are required.
```

Comment:
Could you please add a statement that `embedded` only works if the server of the target url allows embedding via their `CORS` server settings? Together with a rule of thumb `Just test it. If embedded doesn't work for the specified url then you need to use external instead.`.